build/site/specifications/2.1/specs/C2PA_Specification.html

```diff
@@ -760,7 +760,7 @@ <h4 id="_content_credential"><a class="anchor" href="#_content_credential"></a>2
 <p>This is the preferred, non-technical, term for a <a href="#_c2pa_manifest">C2PA Manifest</a>.  The <a href="#_c2pa_manifest_store">C2PA Manifest Store</a> therefore represents the Content Credentials of an asset.</p>
 </div>
 <div class="paragraph">
-<p>Content Credentials also refers to the overall C2PA technology, and is therefore essentially treated as a plural noun. If a <a href="#_c2pa_manifest">C2PA Manifest</a> is a Content Credential, then multiple <a href="#_c2pa_manifest">C2PA Manifest</a> or the broader, universal concept is Content Credentials.</p>
+<p>Content Credentials can also refer to the overall C2PA technology.  If a <a href="#_c2pa_manifest">C2PA Manifest</a> is a Content Credential, then multiple <a href="#_c2pa_manifest">C2PA Manifests</a> or the universally aggreated concepts are Content Credentials.</p>
```

Comment:
Content Credentials ALWAYS refers to the technology - can implies a choice. Also, the grammar is important about the plural noun

What are your concerns with this text?